aip/general/0213.md

```diff
@@ -157,6 +157,37 @@ Occasionally, it may be useful to add protos to these packages or to add to the
 list of commonly-available protos. In order to do this, [open an issue][] on
 the AIP repository in GitHub, noting the guidelines above.
 
+## Existing organization-specific common component packages
```

Comment:
nit: could you add a note stating this is a non-exhaustive list, with a link to where common protos could be found (likely just pointing to the googlapis repo)? I think we should be clear that the common components could be updated separate from this readme.

Reply:
I think that keeping this list up to date _should_ be the goal, and saying it's a non-exhaustive list maybe implies otherwise.

Reply:
I suppose so - but honestly human processes are error-prone. Could we at least add something that points at the googleapis directory as housing these common components?

then if they see one missing at least they know where to look.

Reply:
If there's a canonical place for these libraries to be housed, then I think that's a great idea. My understanding is that each of these would be located in a different place depending on the PA, and so would be difficult to track down.

Reply:
Rather than add anything to this part, I've added two new requirements:

- That org-specific common packages must be published to googleapis
- That orgs must add to this AIP

(Those changes are in a new commit.)

Does that work for both of you?

Reply:
That's great! Thank you

Reply:
thanks Jon, that's fine by me too.